sdk/src/assertions/asset_reference.rs

```diff
@@ -0,0 +1,128 @@
+use serde::{Deserialize, Serialize};
```

Comment:
Every source code file must have the open source copyright at the top. You need to add that here. Copy it from another of the assertions.